config/default.json

```diff
@@ -4,8 +4,8 @@
   "ssl": {
     "enabled": false,
     "port": 443,
-    "cert": "/ssl/cert.pem",
-    "key": "/ssl/key.pem"
+    "key": "./config/key.pem",
```

Comment:
Based on what Kerr said, it sounds like my recommendation of "Use a self-signed cert" is not a good one - i.e. marklogic.maps.arcgis.com wants a real cert, not a self-signed one. I think we can go forward with this for now - I'll test it out locally first to make sure it works for me - but we may need to invest in a real cert and store it somewhere safe, not in version control.